homeassistant/components/drop_connect/binary_sensor.py

```diff
@@ -74,10 +77,23 @@ class DROPBinarySensorEntityDescription(BinarySensorEntityDescription):
         translation_key=PUMP_STATUS,
         value_fn=lambda device: device.drop_api.pump_status(),
     ),
+    DROPBinarySensorEntityDescription(
+        key=ALERT_SENSOR,
+        translation_key=ALERT_SENSOR,
+        device_class=BinarySensorDeviceClass.PROBLEM,
+        value_fn=lambda device: device.drop_api.sensor_high(),
+    ),
+    DROPBinarySensorEntityDescription(
+        key=POWER,
+        translation_key=POWER,
```

Comment:
Did you forget to add this to `strings.json`?

Reply:
No, the power monitoring feature is separate from the `alert_sensor` input.  We intend for the power monitoring sensor to simply be labeled 'Power', so no entry in `strings.json` is necessary.

Reply:
OK, that makes sense. However, if the entity name is not translated, you should not set the `translation_key`.
Maybe something like this, so it's clear from the code it's intentional to not set the name.
```py
    DROPBinarySensorEntityDescription(
        key=POWER,
        translation_key=None,  # Use name provided by binary sensor device class
        device_class=BinarySensorDeviceClass.POWER,
        value_fn=lambda device: device.drop_api.power(),
    ),